src/corgit/corgit.csproj

```diff
@@ -9,4 +9,7 @@
     <RepositoryUrl>https://github.com/jzebedee/corgit</RepositoryUrl>
     <PackageTags>git corgit mingit git-for-windows libgit libgit2 libgit2sharp git.net git# gitsharp</PackageTags>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.2" />
```

Comment:
Took a dependency on `System.Memory` in order to use Span rather than substrings for parsing statuses. This is hopefully a mild change, since:

- We are already locked to `>= netstandard2.0` in order to use `Process`, so there's no issue of losing compatibility by referencing `System.Memory`
- `netstandard2.1` will have Span included, so the package dependency will only affect those targetting `netstandard2.0`